ci/firebase_testlab.py

```diff
@@ -67,6 +69,7 @@ def check_logcat(results_dir):
       'gsutil', 'cat',
       '%s/%s/*/logcat' % (BUCKET, results_dir)
   ])
+  logcat = logcat if isinstance(logcat, str) else logcat.decode(ENCODING)
```

Comment:
Here and in other files: Can this logic go in a function instead of copying it everywhere? That will also make the code easy to update when python2 support is no longer needed.

Reply:
Yeah no problem. I can do that.